plugins/meta/vrf/main.go

```diff
@@ -0,0 +1,164 @@
+package main
```

Comment:
Could you please add copyright section here?

Reply:
Done